src/utils/explorer.ts

```diff
@@ -1,8 +1,4 @@
-const EXPLORER_MAP: { [network: string]: string } = {
```

Comment:
imo this could stay here as it is tightly coupled with explorer.ts .. but it's personal taste, so both ways are probably fine